instructure/canvas-android
# [Flutter-Parent][MBL-13683] Report a problem dialog

This lets app users contact support via the error report endpoint. Added to the ‘contact support’ button on crash screens. Also defaulted our ‘help’ button in the drawer to use this dialog to make it easier for testing. Should not be the default for the ‘help’ dialog in the end game.

To test:
Verify the dialog shows up. If you hit send, it will send an actual error report to canvas support, so probably don't hit send without commenting out the api call. I have done this and will not move the ticket to done until I get a response.
